lab4/src/Search.java

```diff
@@ -0,0 +1,24 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class Search {
+
+    public static void main(String[] args) throws FileNotFoundException {
+        Search search = new Search();
+        search.searchPattern(args[0], args[1]);
+    }
+
+    private void searchPattern(String pattern, String filepath) throws FileNotFoundException {
```

Comment:
"searchPattern" is working in a good way since it is clear and simple to understand